src/block_misc.rs

```diff
@@ -1154,21 +1154,21 @@ mod tests {
         // Order depends on the input parameters order, not on order of adding to graph
         assert_eq!(pb.observations, vec![6.0, 7.0, 1.0, 2.0, 3.0, 4.0, 5.0]); // join actually doesn't do anything
 
-        spredict2(&mut bg, &fb, &mut pb, false);
+        spredict2(&mut bg, &fb, &mut pb);
         assert_eq!(pb.observations, vec![6.0, 7.0, 1.0, 2.0, 3.0, 4.0, 5.0]); // join actually doesn't do anything
```

Comment:
Duplicated comment, perhaps a general one before this test suffices

Reply:
Added to comment on line 1154.